src/modelAnalysis/FBA/optimizeCbModel.m

```diff
@@ -84,7 +84,7 @@
 %    FBAsolution:       solution object
 %
 %                          * f - Objective value
-%                          * x - Primal
+%                          * v - Reaction rates (Optimal primal variable, legacy FBAsoltion.x)
 %                          * y - Dual
 %                          * w - Reduced costs
 %                          * s - Slacks

```

Comment:
Why this change to v ? Almost all code I know relies on x. Whats the reason for this change?